### ISSUE TITLE
Font Awesome Icons Can't Load Cross-Domain

### ISSUE THREAD

**Issue by DeedleFake** [alanorth/hugo-theme-bootstrap4-blog#23]
I've got a site set up behind a reverse proxy that forwards it requests to both `deedlefake.com` and `www.deedlefake.com`, as is the convention. In the Hugo config, I've configured `https://deedlefake.com` as the base URL. For some reason, although the rest of the font seems to load fine, the icons for categories and tags don't load if viewing the site through www.deedlefake.com.

This may be a configuration issue on my end, but, since I haven't been able to figure out a way to fix it, I'm filing this in case it isn't.

I suppose the other option would just be to have the `www` subdomain redirect, rather than attempting to show both, but I'd like to be able to if I can.

**Comment by fte378** [comment]
Just tried it and the icons are there also on www.deedlefake.com.

**Comment by DeedleFake** [comment]
Odd. On my Windows, Linux, and Android systems I see something like what's in the attached screenshot, which was taken with a Linux system. Just broken Unicode characters, if anything displays at all.

![screenshot from 2016-11-30 13-27-17](https://cloud.githubusercontent.com/assets/326750/20765572/e8328d62-b700-11e6-9d91-3e32d16c95ae.png)

**Comment by alanorth** [comment]
This is a web server configuration issue. Chrome dev tools says:

> Access to Font at 'https://deedlefake.com/fonts/fontawesome-webfont.woff2?v=4.7.0' from origin 'https://www.deedlefake.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin 'https://www.deedlefake.com' is therefore not allowed access.

Also, I would recommend that you choose one domain name and stick to it. First, the "www" convention is very old and I've been a "no WWW" supporter for at least ten years. Second, the real issue is that if you provide the same content on your non-WWW and WWW domains, then Google will see these as two separate sites, and you'll essentially be competing with yourself for content/tags/keywords in the search results. This is called the [_duplicate content problem](https://support.google.com/webmasters/answer/66359?hl=en).

So basically, you need to set your web server to redirect WWW→no-WWW with an HTTP 301 or 302 at the virtual host level.
